### PR TITLE
build: Change Windows master build to use raw Discord webhook

### PR DESCRIPTION
**Pull request by adamdmharvey**
## Hey, I just made a Pull Request!

Noticed the master builds were failing on the Windows step, like so:

image
https://github.com/backstage/backstage/runs/1596622881

As a hack, this PR just converts it to use a raw string for posting the alerts out since most of the packaged Actions don't support Windows. Tested on my private repo and the mechanics worked, assuming the discord hook (same as used in Linux builds) is fine!

image

#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [x] Screenshots attached (for UI changes)

### CODE REVIEW

**Review by Rugvip — APPROVED** [review]
Thank you! Let's try it :grin:

**Comment by adamdmharvey** [comment]
So how should your new "friends" work with everyone - in cases like this, you've approved my PR, should I wait for a core maintainer to merge since it affects the build process, or any guidelines as I learn the ropes?

image

**Comment by Rugvip** [comment]
@adamdmharvey Fine to merge if approved! In general we prefer having the creator of PRs merge them, just in case there are some pending fixes or what have you. Sometimes you'll might get an approve + comments, in which case any simple comments are expected to be addressed while larger things would be for followups or later work.

**Comment by freben** [comment]
Maybe we should write a little playbook.

**Comment by dtuite** [comment]
I find it helps to close your eyes while you press the merge button.

**Comment by adamdmharvey** [comment]
Good idea, @freben. If I get a chance in the next week or so, I'll throw some words into `CONTRIBUTING.md` and others are welcome to `:achem`: contribute or interate.

**Comment by adamdmharvey** [comment]
...and here we go

**Comment by adamdmharvey** [comment]
I guess it worked ->

image

Though `master` peppered with some ❌ errors to address.

**Pull request merged**
